### ISSUE TITLE
ACCEPT prints a space in the middle of very long strings

### ISSUE THREAD

**Issue by SamCoVT** [scotws/TaliForth2#70]
It looks like ACCEPT prints a space when it reaches the end of a line of input, regardless of whether it's because of a AscLF or AscCR, or if it's because it completely filled the buffer.  Very long strings call accept multiple times (via REFILL).  Every time 255 characters (the size of the input buffer) have been read, it automatically prints a space.  The space doesn't end up in the final string, but it looks weird on the screen.

I recommend having ACCEPT not print the space if it completely fills the buffer.  The standard actually recommends no space even for a line-terminator, so perhaps that space should just be removed.  See the last paragraph of the Rationale section for ACCEPT:
[Forth 2012 ACCEPT](http://lars.nocrew.org/forth2012/core/ACCEPT.html)

Removing it from `_eol:` should do the trick.

**Comment by SamCoVT** [comment]
I tried commenting the space out from `_eol:` in ACCEPT, just to see what would happen.

```
Type 'bye' to exit
1 2 + .3  ok
       ^ enter pressed here
variable stuff 10 allot ok
stuff 10 acceptabcdefghij ok
               ^ enter pressed here
```

I'm pretty sure I walked through a tutorial with accept in it and it talked about a space being there.
Gforth has a space in its accept, but gforth accept interestingly just stops the cursor after the requested number of characters has been read (extra letters are ignored) and forces you press ENTER to finish.  I don't think we want that particular behavior in Tali.  I like how the standard just leaves a lot of this as "implementation-defined".

I think I'm going to have it print the space for line terminators, but not for full-buffer termination.  I think that makes the most sense.
